.editorconfig

```diff
@@ -1,7 +1,1208 @@
-[*.{js,jsx,ts,tsx,vue}]
-indent_style = space
-indent_size = 2
+[*]
```

Comment:
Thanks for updating this

Reply:
These can actually be generated from IntelliJ, so that's exactly what I did. I made one that matches our standards, and exported it as `.editorconfig`